src/routes/activityReports/index.js

```diff
@@ -40,6 +41,7 @@ router.post('/', transactionWrapper(createReport));
 router.get('/approvers', transactionWrapper(getApprovers));
 router.get('/groups', transactionWrapper(getGroups));
 router.get('/activity-recipients', transactionWrapper(getActivityRecipients));
+router.get('/activity-recipients/:reportId', transactionWrapper(getActivityRecipientsForExistingReport));
```

Comment:
might be a useless change, but how do we feel about flipping this to be``` /:reportId/activity-recipients``` (i.e ```/api/activity-reports/1/activity-recipients```)

Reply:
Yeah.. I had this same thought. I'll make this change 👍